### PR TITLE
docs: fix docstring rankdriver

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

File: jina/drivers/rank/aggregate/__init__.py [hunk 1]
```diff
@@ -149,10 +149,6 @@ class Chunk2DocRankDriver(BaseAggregateMatchesRankerDriver):
                 |    |- matches: {granularity: k}
                 |
                 |-matches: {granularity: k-1} (Ranked according to Ranker Executor)
-
-        .. note::
```
Comment (Yongxuanzhang) [inline jina/drivers/rank/aggregate/__init__.py:153]: Why remove this. It's not correct?

Comment (JoanFM) [reply jina/drivers/rank/aggregate/__init__.py:153]: It changed to traverse on root.

Comment (Yongxuanzhang) [reply jina/drivers/rank/aggregate/__init__.py:153]: To my understanding, for this driver, we pass in the document, the document has several chunks, and chunks have matches. Then we traverse all the chunks of the document to do some ranking(score). Is it correct? 
If so, is it a traversal on chunks or root?
If not, what would be the case?

Comment (JoanFM) [reply jina/drivers/rank/aggregate/__init__.py:153]: This is correct, therefore the docs in input of the apply_all is the root

Comment (Yongxuanzhang) [reply jina/drivers/rank/aggregate/__init__.py:153]: 😂  Ok. I think it's fine to remove it. The docstring above is clear and understandable to me.

**Review by Yongxuanzhang — APPROVED** [review]

**Pull request merged**